dev-tools/mage/pkgdeps.go

```diff
@@ -117,22 +120,64 @@ func installDependencies(arch string, pkgs ...string) error {
 		}
 	}
 
-	if err := sh.Run("apt-get", "update"); err != nil {
-		return err
+	err := fixJessieRepositories()
+	if err != nil {
+		return fmt.Errorf("error while editing the repositories: %w", err)
 	}
 
-	params := append([]string{"install", "-y", "--force-yes",
-		"--no-install-recommends",
+	// TODO: This is only for debian 7 and should be removed when move to a newer OS. This flag is
+	// going to be used unnecessary when building using non-debian7 images
+	// (like when making the linux/arm binaries) and we should remove it soonish.
+	// See https://github.com/elastic/beats/v7/issues/11750 for more details.
```

Comment:
URL is broken (remove `v7/`)

Reply:
Thanks! Will do, it was copy pasted from here https://github.com/elastic/beats/blob/2d32a6e6b541ce8b5a7c377e99bc0243911b40ab/x-pack/auditbeat/magefile.go#L221